src/plugins/intel_gpu/src/graph/include/program_node.h

```diff
@@ -196,6 +196,9 @@ struct program_node {
     void remove_dependency(size_t idx);
     void remove_dependency(program_node& node);
 
+    size_t get_dependency_index(program_node& node);
+    size_t get_user_index(program_node& node);
```

Comment:
these methods should be const I believe.

Reply:
Applied.